### PR TITLE
AutoStake rebrand from .net to .com and add missing services

### PR DESCRIPTION
**Pull request by AutoStake-com**

### CODE REVIEW

**Review by dylanschultzie — COMMENTED** [review]
Looks fine to me as well...

**Comment by AutoStake-com** [comment]
Now it's ok with Rebus and complaining elsewhere ¯\_(ツ)_/¯

The test seems to have some kind of bug

File: sifchain/chain.json [hunk 1]
```diff
@@ -94,6 +94,10 @@
         "address": "https://rpc-sifchain-ia.cosmosia.notional.ventures/",
         "provider": "Notional"
       },
+      {
+        "address": "https://sifchain-mainnet-rpc.autostake.com:443",
+        "provider": "AutoStake 🛡️ Slash Protected"
+      }
```
Comment (JeremyParish69) [inline sifchain/chain.json:100]: here needs a comma

Comment (AutoStake-com) [reply sifchain/chain.json:100]: OMG, thank you so much!

**Comment by AutoStake-com** [comment]
So the error was just misleading and infact it was indeed my own fault!

Thank you both

**Pull request merged**